tests/e2e/execution-test.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#     Copyright 2021 Dynatrace LLC
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+readonly EXPECTED_NUMBER_OF_LOGS=3
+
+echo "Waiting for Logic app to produce logs for 3 min..."
+START_TIME=$(date +%s%N | cut -b1-13)
+sleep 3m
+END_TIME=$(date +%s%N | cut -b1-13)
+echo "Try to receive logs from Dynatrace in 5 min (timeframe: start time=$START_TIME, end time=$END_TIME)"
+sleep 5m
+
+readonly SEARCH_QUERY='query=azure.resource.type="MICROSOFT.LOGIC/WORKFLOWS/RUNS/TRIGGERS" AND content="\"status\": \"Running\""'
```

Comment:
is it possible to query for triggers from specific LogicApp instance? to prevent false positive caused by other instance reporting other logicapp logs to same tenant?